### DISCUSSION TITLE
Shopclass Theme

### DISCUSSION THREAD

**Discussion by eurobank** [mindstellar/shopclass#461] [Q&A] [answered]
I would like to ask if Shopclass theme works with the latest Osclass 5.x version.

**Comment by navjottomer** [comment, marked as answer]
If you find any issue then let me know. Currently, I do not have enough
time to test this that's why I did not created any release.
But my site tuffclassified.com is running on latest Osclass and
Shopclass theme with minor changes. It is possible that I forgot to include
those changes in repository version.


On Thu, 23 Dec 2021 at 21:43, eurobank ***@***.***> wrote:

> I would like to ask if Shopclass theme works with the latest Osclass 5.x
> version.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mindstellar/Osclass/discussions/461>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIDQSMQ7Y6Z7LSO56SSWKDUSNDA5ANCNFSM5KVCSNMQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Regards
Navjot Tomer

**Comment by eurobank** [comment]
Got it. I will report any issues there.
